libcnb-proc-macros/Cargo.toml

```diff
@@ -2,6 +2,12 @@
 name = "libcnb-proc-macros"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.56"
+license = "BSD-3-Clause"
+description = "Procedural macros used within libcnb.rs"
+repository = "https://github.com/Malax/libcnb.rs"
+readme = "README.md"
```

Comment:
This points to `libcnb-proc-macros/README.md`, but no such file exists at the moment.

Reply:
Ooof. Yeah.